controls/basics/how-to-git.md

```diff
@@ -1,12 +1,33 @@
-## How to use git
+## What is git and what is it used for? 
 
-### What is git
+<img src="../../assets/images/git-logo.png" alt="git logo" style="width:150px; float: right; padding: 25px;">
+Git is something called a content tracker which is capable of cataloging all changes you make to tracked files. Git centers around the manipulation of branches of code, the merging and converging of existing code branches to improve a ending product. When you create or edit a file on your computer, the changes you make locally will be recorded and can differ from that of the global codebase otherwise known as the main branch. This dissonance provides greater quality control to the maintainer (permitting these changes), and allows for a more adaptable development cycle for the programmer through quality of life commands such as: "git revert", "git diff", and "git merge" How all of these benefits are achieved is through proper utilization.
 
-### What is git used for
+### How to use git
+
+Commonly Used Commands:
+<ul>
+    <li>git add %filename%</li>
+    The git add command adds a filename to the tracker and its changes will be recorded by git. Only files added through the git add command will be included in commits and pushes later down the line.
```

Comment:
surround all things that would be put into a command line with backticks `like this`, provides easy dilneation as to what parts are the commands.

```suggestion
    The `git add` command adds a filename to the tracker and its changes will be recorded by git. Only files added through `git add` will be included in commits (and by implication, pushes) later down the line.
```